CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.7.1
 - Version 0.7.0 broke most of the `message_detail_level` settings, only `basic` worked. 0.7.1 fixes it.
+- Added `send_organization` configuration which provides the optional ability to include the organization in /etc/chef/client.rb to Slack.
```

Comment:
`0.7.1` has already been released to this should go in an `Unreleased` section instead